TSC-Charter.md

```diff
@@ -36,18 +36,19 @@ approved by the Board.
 
 ## Section 4. Establishment of the TSC.  
 
-TSC memberships are not time-limited. There is no fixed size of the TSC.
-However, the expected target is between 6 and 12, to ensure adequate
-coverage of important areas of expertise, balanced with the ability to
-make decisions efficiently.
+TSC memberships are not time-limited. There is no maximum size of the TSC.
+The size is expected to vary in order to ensure adequate coverage of important
+areas of expertise, balanced with the ability to make decisions efficiently.
+The TSC must have at least three members.
```

Comment:
Indeed, it's exceedingly unlikely. If the membership were to even get close to falling below that number, I would suspect that the separate CTC and TSC bodies would remerge into one. If *that* were to fall below 3 members, then we'd really be in trouble.